src/libraries/System.Private.CoreLib/src/System/Globalization/CompareInfo.cs

```diff
@@ -1599,5 +1615,9 @@ public SortVersion Version
         }
 
         public int LCID => CultureInfo.GetCultureInfo(Name).LCID;
+
+#if TARGET_BROWSER
+        private static string GetPNSEText(string funcName) =>  $"{funcName} is not supported when HybridGlobalization=true. Disable it to load larger ICU bundle, then use this option.";
```

Comment:
Should this be in resource file and localized ?

Reply:
Good idea, I can see all exception text in PrivateCoreLib are stored like this, so you are right.